UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/ObjectLayer.cs

```diff
@@ -148,7 +118,10 @@ public override void ClearAllTiles()
 				DestroyImmediate(obj.gameObject);
 			}
 		}
+	}
+
+	public override void RecalculateBounds()
```

Comment:
Codacy found an issue: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6319358)